src/components/Article.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.scss';
+
+export const Article = ({
+  title = 'No Name',
+  text = 'No description',
+  date = 'No information',
+}) => (
```

Comment:
you've already added defaultProps

```suggestion
export const Article = ({ title, text, date }) => (
```